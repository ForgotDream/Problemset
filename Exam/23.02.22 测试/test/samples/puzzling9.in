5
5 3
100
100
110
010
011
3 3
111
011
011
3 2
10
11
01
2 3
111
011
2 1
1
1
