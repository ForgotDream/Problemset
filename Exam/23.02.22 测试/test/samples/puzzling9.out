13222
13322
11322
51444
51144